app/Http/Controllers/CategoriesController.php

```diff
@@ -9,7 +9,13 @@ class CategoriesController extends Controller
 {
     public function index()
     {
-        $categories = Category::all();
+        $categories = collect([]);
+        $records = Category::all();
```

Comment:
you can directly use map api in Collection for $records, no need to to loop push in new collection

Reply:
reference: https://laravel.com/docs/5.8/collections#method-map

Reply:
I have resolved this in 2a514d2